python/kfserving/kfserving/api/kf_serving_client.py

```diff
@@ -24,10 +24,16 @@
 
 
 class KFServingClient(object):
-    '''KFServing Client Apis.'''
 
     def __init__(self, config_file=None, context=None, # pylint: disable=too-many-arguments
                  client_configuration=None, persist_config=True):
+        """
+        KFServing client constructor
+        :param config_file: kubeconfig file, defaults to ~/.kube/config
+        :param context:
```

Comment:
Really great to add the docString for this, I'm thinking if that'e better to add the description for those param (some of them already have), since I'm working for kubeflow-fairing API docs, that's good to generate that if have description for the param.